patterns/about-details-description-with-bullets.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Title: About Details Description
+ * Slug: twentytwentyfive/about-details-description
+ * Title: About - Detailed description with bullets
+ * Slug: twentytwentyfive/about-details-description-with-bullets
```

Comment:
There should only be one slug,

Reply:
Thanks for the Feedback.

There was a conflict with a previous implementation in the branch, but it has been resolved. I have removed the duplicate code to address the issue.